prov/rxd/src/rxd_ep.c

```diff
@@ -506,9 +506,17 @@ void rxd_ep_free_acked_pkts(struct rxd_ep *ep, struct rxd_x_entry *x_entry,
 int rxd_ep_retry_pkt(struct rxd_ep *ep, struct rxd_pkt_entry *pkt_entry,
 		     struct rxd_x_entry *x_entry)
 {
-	return fi_send(ep->dg_ep, (const void *) rxd_pkt_start(pkt_entry),
-		       pkt_entry->pkt_size, rxd_mr_desc(pkt_entry->mr, ep),
-		       x_entry->peer, &pkt_entry->context);
+	int ret;
+
+	ret = fi_send(ep->dg_ep, (const void *) rxd_pkt_start(pkt_entry),
+		      pkt_entry->pkt_size, rxd_mr_desc(pkt_entry->mr, ep),
+		      x_entry->peer, &pkt_entry->context);
+
+	if (ret)
+		FI_WARN(&rxd_prov, FI_LOG_EP_CTRL,
+			"Error sending message\n");
```

Comment:
Can fi_send with EAGAIN, which would be a common occurrence?  If so, this may be better as FI_INFO.  I would also display the reason for the error (fi_tostr(-ret))